### DISCUSSION TITLE
Test-DbaLastBackup: Add -FileStreamDirectory (similarly with Restore-DbaDatabase)

### DISCUSSION THREAD

**Discussion by christophorosp** [dataplat/dbatools#7847] [Ideas]
First off, congrats about the project; dbatools is an amazing set of tools for the DBA out there

Second, thanks for all the support here as well as in your Slack channel; y'all rock.

Lastly, here's my question:
As per title, is it feasible to differentiate DataDirectory from FileStreamDirectory and introduce a new parameter (-FileStreamDirectory ) like the one on Restore-DbaDatabase (-DestinationFileStreamDirectory) command?

**Comment by andreasjordan** [comment]
That would be a feature request. Just had a look at the code and it should not be to much work as all these parameters are just passed to Restore-DbaDatabase. Please open a feature requst here: https://github.com/sqlcollaborative/dbatools/issues/new?assignees=&labels=feature%2Ctriage+required&template=feature-request.yml
